daemon/cmd/status.go

```diff
@@ -793,6 +793,41 @@ func (d *Daemon) startStatusCollector(cleaner *daemonCleanup) {
 				}
 			},
 		},
+		{
+			Name: "config-settings",
```

Comment:
How confident are we that our logic never produces any false positives? Should we add a flag to disable this mechanism in case it does produce false positives and users want to turn the warning off?

Reply:
If the assertions above are correct we can't guarantee we will be 100% on with our recommendations ;(

Reply:
Maybe we could introduce an agent flag that disables reporting for certain flags? I'm thinking of something along the lines of `--config-ignore-drift=ipam,enable-wireguard" which would silence drift warnings for the "ipam" and "enable-wireguard' flags.

This would be a "trust me I know what I'm doing" kind of option for folks who are doing, or as a escape hatch in case we do produce false positives (which of course we would want to fix, but users might still want to silence such warnings while they wait for the patch to be released)